### PR TITLE
WIP: implement creation of session for an account

### LINKED ISSUE #217: Implement /accounts/:id/sessions post route
In order admins to create a session for a certain account the POST route needs to be implemented.

### PR DESCRIPTION
**Pull request by barretodaniel**
closes #217

### CODE REVIEW

File: routes/accounts.js [hunk 1]
```diff
@@ -270,6 +270,44 @@ function accountRoutes (server, options, next) {
     }
   }
 
+  var postAccountsSessionRoute = {
```
Comment (barretodaniel) [inline routes/accounts.js]: I've started this in the accounts routes file since it's related. I'm wondering if I should create a separate file for this.

Comment (gr2m) [inline routes/accounts.js]: I’d create a new file for it. I’d suggest `routes/accounts/sessions.js` :)

File: routes/accounts.js [hunk 2]
```diff
@@ -270,6 +270,44 @@ function accountRoutes (server, options, next) {
...
+    handler: function (request, reply) {
+      var username = request.payload.data.attributes.username
+      var password = request.payload.data.attributes.password
+      var query = request.query
+
+      var sessionId = toSessionId(request)
+
+      admins.validateSession(sessionId)
+
+      .then(function (_) {
```
Comment (barretodaniel) [inline routes/accounts.js]: previous examples show this having a parameter `doc` but its not used. Should I do anything with the admin doc before returning the result of the session add?

Comment (gr2m) [inline routes/accounts.js]: if you don’t do anything with the argument, leave it out entirely (`.then(function () {`) or comment it out (`.then(function (/* doc */) {`)

**Pull request closed without merging**